src/components/Footer.styled.tsx

```diff
@@ -0,0 +1 @@
+
```

Comment:
Would you be able to create a style for the footer so that the footer sticks to the bottom of the page?